### PR TITLE
JBTIS-4.2.4.Final/ JBDSIS 8.0.4.GA updates

### PR DESCRIPTION
**Pull request by pleacu**
Update product info for JBTIS-4.2.4.Final/ JBDSIS 8.0.4.GA.

### CODE REVIEW

**Comment by pleacu** [comment]
@xcoulon - does this look okay to push?  Thkx!

File: _config/products.yml [hunk 1]
```diff
@@ -540,28 +551,6 @@ devstudio_is:
             file_size: 294MB
             url: https://devstudio.jboss.com/updates/8.0.0/jbdevstudio-integration-stack-updatesite-8.0.0.GA.zip
             md5_url: https://devstudio.jboss.com/updates/8.0.0/jbdevstudio-integration-stack-updatesite-8.0.0.GA.zip.MD5
-      8.0.0.Beta2:
```
Comment (xcoulon) [inline _config/products.yml]: why are you removing these older entries ? We keep all previous versions, including Alpha/Beta/CR

File: _config/products.yml [hunk 2]
```diff
@@ -1510,28 +1508,6 @@ jbt_is:
             file_size: 214MB
             url: http://download.jboss.org/jbosstools/updates/stable/luna/integration-stack/aggregate/jbosstools-integration-stack-aggregate-4.2.0.Final.zip
             md5_url: http://download.jboss.org/jbosstools/updates/stable/luna/integration-stack/aggregate/jbosstools-integration-stack-aggregate-4.2.0.Final.zip.MD5
-      4.2.0.Beta2:
```
Comment (xcoulon) [inline _config/products.yml]: Same comment here: why are you removing those entries ?

**Comment by xcoulon** [comment]
Updated the blog entry date and merged to master.

**Pull request closed without merging**